package.json

```diff
@@ -13,8 +13,8 @@
   "scripts": {
     "build": "concurrently \"cd packages/frontend && npm run build\" \"cd packages/backend && npm run build\"",
     "watch": "concurrently \"cd packages/frontend && npm run watch\" \"cd packages/backend && npm run watch\"",
-    "format:check": "prettier --check \"**/src/**/*.{ts,svelte}\"",
-    "format:fix": "prettier --write \"**/src/**/*.{ts,svelte}\"",
+    "format:check": "biome format",
```

Comment:
markdown files won't be formatted/checked

Reply:
Yes, that's unfortunately. I will likely put up another PR to start using prettier for md the same as podman-desktop repo. (checked the biome roadmap, they're hoping to add md this year but it is well down the list)